clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -123,8 +123,8 @@ private void checkQuotas(long timeMs) {
                         throw new QuotaViolationException(String.format(
                             "(%s) violated quota. Actual: (%f), Threshold: (%f)",
```

Comment:
The usage of `()` to delineate values is a bit weird. Maybe we can replace it with `'` or something like that.